cityofaustin/atd-vz-data
# Ability to create temporary crashes in the new data model

## Associated issues
Closes https://github.com/cityofaustin/atd-data-tech/issues/18039

## Testing
**URL to test:** 
Local

**Steps to test:**
1. Check out this branch and follow instructions in https://github.com/cityofaustin/atd-vz-data/pull/1440 to start the data model, apply migrations and run the CRIS import script
2. Go to the create temp crash page under Tools in the sidebar
3. Note that we now have a crash timestamp field and the address fields auto capitalize your input
4. Test adding temp crashes with various combos of units and fatalities/injuries
5. When your temp crash is successfully created, a banner will pop up at the top with a link to it
6. Temp crashes will have a `record_locator` value of `T<id>` normal crashes will have a `record_locator` value of just their `<id>`
7. This new `record_locator` field is what is now used for routing
8. In the URL if ur lazy like me u dont need to uppercase the "T", the id will be converted to upper case before querying the database so "http://localhost:3000/#/crashes/t32449" for example will work
9. Test going to the temp records crash details page, editing the units, people, etc. Make sure everything is working

---
#### Ship list
